test/test_corpus.cc

```diff
@@ -132,6 +132,56 @@ unique_ptr<Diagnostic> errorToDiagnostic(const core::GlobalState &gs, const core
     return make_unique<Diagnostic>(Range::fromLoc(gs, error.loc), error.header);
 }
 
+class ErrorHandler {
+    vector<unique_ptr<core::Error>> &errors;
```

Comment:
Is there a reason why `ErrorHandler` doesn't own `errors`? Seems like a footgun to store a reference to it.